source/triggers/database-triggers.txt

```diff
@@ -137,81 +151,141 @@ Database Triggers have the following configuration options:
-
-.. list-table::
-   :header-rows: 1
-   :widths: 15 30
-
-   * - Field
-     - Description
-
-   * - :guilabel:`Cluster Name`
-
-     - The name of the MongoDB Service that the Trigger is
-       associated with.
+Trigger Source Details
+~~~~~~~~~~~~~~~~~~~~~~
 
-   * - :guilabel:`Database Name`
+Within the :guilabel:`Trigger Source Details` section, you first select the 
+:guilabel:`Source Type`, based on the level of granularity you want. Your 
+options are:
 
-     - The MongoDB database that contains the watched
-       collection.
+- :guilabel:`Collection`, when a change occurs on a specified collection
+- :guilabel:`Database`, when a change occurs on any collection in a 
+  specified database
+- :guilabel:`Deployment`, when deployment changes occur on a specified 
+  cluster. If you select the Deployment source type, the following 
+  databases are **not watched** for changes:
 
-   * - :guilabel:`Collection Name`
+  - The admin databases ``admin``, ``local``, and ``config``
+  - The sync databases ``__realm_sync`` and ``__realm_sync_<app_id>``
 
-     - The name of the collection that the Trigger watches for
-       change events.
+Depending on which source type you are using, the additional options differ. The 
+following table describes these options.
 
-   * - :guilabel:`Operation Type`
+.. list-table::
+   :header-rows: 1
+   :widths: 15 30
 
-     - Checkboxes of :ref:`database operation types <database-event-operation-types>` 
-       that cause the Trigger to fire. Select the operation types you want the 
-       trigger to respond to.
+   * - Source Type
+     - Options
 
-       .. warning::
+   * - :guilabel:`Collection`
+     - | 
 
-          Update operations executed from MongoDB Compass or the MongoDB Atlas
-          Data Explorer fully replace the previous document. As a result,
-          update operations from these clients will generate **Replace**
-          change events rather than **Update** events.
+       - :guilabel:`Cluster Name`. The name of the MongoDB cluster that the 
+           Trigger is associated with.
+      
+       - :guilabel:`Database Name`. The MongoDB database that contains the watched
+         collection.
 
-   * - :guilabel:`Full Document`
+       - :guilabel:`Collection Name`. The MongoDB collection to watch. Optional.
+         If you leave this option blank, the Source Type changes to "Database."
 
-     - If enabled, **Update** change events include the latest
-       :manual:`majority-committed </reference/read-concern-majority/>`
-       version of the modified document *after* the change was applied in
-       the ``fullDocument`` field.
+       - :guilabel:`Operation Type`. The collection operation types 
+         that cause the Trigger to fire. Select the operation types you want the 
+         trigger to respond to. Options include:
+       
+         - Create Document
+         - Modify Document
+         - Rename Document
+         - Shard Document
+         - Drop Document
```

Comment:
This should be Insert, Update, Replace, and Delete

Reply:
Doh!